data-access/basic_data_access.ipynb

```diff
@@ -167,538 +248,1117 @@
    },
    "outputs": [],
    "source": [
-    "get_data_options(\n",
-    "    variable = \"precipitation\", \n",
-    "    timescale = \"daily\", \n",
-    "    tidy = False\n",
-    ") "
+    "cd.reset()\n",
+    "print(\"Interface reset - ready for new query\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5501a518",
+   "metadata": {},
+   "source": [
+    "# Retrieving data \n",
+    "The `ClimateData` interface allows you to chain method calls to build readable queries, and then retrieve the data easily in your query. \n",
+    "<br><br>\n",
+    "Required components of the query depend on the data catalog you're interested in. In general, the required components for all queries are: \n",
+    "- catalog\n",
+    "- variable "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "99bb1909",
+   "metadata": {},
+   "source": [
+    "### Example 1: Future air temperature data \n",
+    "You can retrieve data using a dictionary query, or by chaining operations to the `ClimateData` object. Either is valid and will result in the same output data, so just use whichever method is most intuitive to you. "
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "f3f954d4-7a75-4dcd-b6d6-8e7d730fc607",
+   "id": "7d827595",
    "metadata": {},
    "source": [
-    "## See all the geometry options for spatially subsetting the data during retrieval\n",
-    "These options will match those in our AE selections GUI. This will enable you to retrieve a subset for a specific region."
+    "#### Method 1: Chained operations (recommended)"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "ddc1fdd4-8bc2-4785-ace1-229ba168d54f",
-   "metadata": {
-    "tags": []
-   },
+   "id": "f2b4dfd3",
+   "metadata": {},
    "outputs": [],
    "source": [
-    "get_subsetting_options()"
+    "cd.reset()\n",
+    "climate_data = (cd\n",
+    "    .catalog(\"cadcat\")\n",
+    "    .activity_id(\"WRF\")\n",
+    "    .experiment_id(\"ssp370\")\n",
+    "    .institution_id(\"UCLA\")\n",
```

Comment:
I remember being told to use `institution_id('UCLA')` whenever I'm accessing WRF - can we add a note about that in the notebook, if that's still important?